docs/faq.rst

```diff
@@ -121,6 +121,13 @@ tox 4 -- output changes
 - We now use colors for reporting, to help make the output easier to read for humans. This can be disabled via the
   ``TERM=dumb`` or ``NO_COLOR=1`` environment variables, or the ``--colored no`` CLI argument.
 
+tox 4 -- re-use of environments
++++++++++++++++++++++++++++++++
+
+- It is no longer possible to re-use environments. While this was possible with tox version 3, this behavior was never
```

Comment:
It could be beneficial to mention that the reason this is not supported is that:

```ini
[testenv]
envdir = .tox/venv

[testenv:a]
deps = pytest>7

[testenv:b]
deps = pytest<7
```

It would create an environment that does not have its dependencies correctly specified.